WebRoot/skins/_base/base4/skin.css

```diff
@@ -1066,3 +1066,41 @@ div.ZmHtmlEditor .TxtAreaDrag {
 .ZmConvDoublePaneView .Row-selected .ZmMsgListDate {
     background-color: @ListItemBgColor-selected@ !important;
 }
+
+/*Circular Progress bar styling*/
+svg.progress-circle {
+	@FullWidth@
+}
+
+svg.progress-circle text {
+	fill: @TxtC@;
+}
+
+svg.progress-circle circle.bg {
+	stroke: #e5e5e5;
```

Comment:
Done: https://github.com/Zimbra/zm-web-client/pull/63/commits/a7b2fcdc7e6d460347cdbfc70cafc577ab173260#diff-6d758c79df985840797b946711380390R1080
and https://github.com/Zimbra/zm-web-client/pull/63/commits/a7b2fcdc7e6d460347cdbfc70cafc577ab173260#diff-16ae02866e4f63813de9a7825a1057e3R2085